### PR TITLE
Use Mry protobuf codec for the whole Database service

### PR DESCRIPTION
**Pull request by mlaflamm**

### CODE REVIEW

File: mry-core/src/main/scala/com/wajam/mry/ConsistentDatabase.scala [hunk 1]
```diff
@@ -68,6 +68,11 @@ class ConsistentDatabase[T <: ConsistentStorage](serviceName: String = "database
           val transaction = new Transaction()
           value.applyTo(transaction)
           val message = new InMessage(Map(Database.TOKEN_KEY -> value.token), data = transaction)
+          message.token = value.token
+          message.serviceName = name
+          message.method = ActionMethod.POST
+          message.function = MessageType.FUNCTION_CALL
+          message.path = remoteWriteExecuteToken.path.buildPath(message.parameters)
```
Comment (oliviertoupin) [inline mry-core/src/main/scala/com/wajam/mry/ConsistentDatabase.scala:75]: Why those additions?

Comment (mlaflamm) [inline mry-core/src/main/scala/com/wajam/mry/ConsistentDatabase.scala:75]: I am recreating MRY write messages from the database data. The most important thing is fill in the message data with the Transaction (was obviously already done). This is not critical to also include the metadata but this helps debugging very much.

Comment (oliviertoupin) [inline mry-core/src/main/scala/com/wajam/mry/ConsistentDatabase.scala:75]: ok

**Comment by oliviertoupin** [comment]
+1

**Pull request merged**